inc/sainexthop.h

```diff
@@ -45,11 +45,64 @@ typedef enum _sai_next_hop_type_t
     SAI_NEXT_HOP_TYPE_MPLS,
 
     /** Tunnel next hop */
-    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP
+    SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP,
+
+    /** IPv6 Segment Route SID List */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_SIDLIST,
+
+    /** IPv6 Segment Route Endpoint Function */
+    SAI_NEXT_HOP_TYPE_SEGMENTROUTE_ENDPOINT
 
 } sai_next_hop_type_t;
 
 /**
+ * @brief Enum defining Endpoint Action types
```

Comment:
I'm confused on this.  It should be apparent from the enum list, which corresponds to the list in https://tools.ietf.org/html/draft-filsfils-spring-srv6-network-programming-00#section-4